eng/pipelines/libraries/helix.yml

```diff
@@ -18,9 +18,10 @@ steps:
             /t:test
             /p:RuntimeFlavor=${{ parameters.runtimeFlavor }}
             /p:ArchGroup=${{ parameters.archGroup }}
-            /p:ConfigurationGroup=${{ parameters.configuration }}
+            /p:Configuration=${{ parameters.configuration }}
             /p:OSGroup=${{ parameters.targetOS }}
             /p:TestScope=${{ parameters.testScope }}
+            /p:BuildTargetFramework=${{ parameters.framework }}
```

Comment:
Why BuildTargetFramework and not TargetFramework?

You shouldn't do this here.

We pass down the TargetFramework via `extraHelixArguments`.

https://github.com/dotnet/runtime/blob/23c54f183c9faf2049223a44197c6b8b282b90c4/eng/pipelines/libraries/base-job.yml#L74
https://github.com/dotnet/runtime/blob/23c54f183c9faf2049223a44197c6b8b282b90c4/eng/pipelines/libraries/base-job.yml#L78

Please remove this and if we instead need to pass `BuildTargetFramework` do it directly in the place I linked above and add a comment of why `TargetFramework` is not sufficient.